docusaurus.config.js

```diff
@@ -1,52 +1,43 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
-
 import { themes as prismThemes } from 'prism-react-renderer';
-
 const lightCodeTheme = prismThemes.github;
 const darkCodeTheme = prismThemes.dracula;
-
```

Comment:
Why are you removing all the empty lines? They visually separate different pieces of code.

Reply:
I used a formatter and did not see that. I will correct it .